pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
   "albumentations",
   "click",
   "clip_anytorch",          # replacing "clip @ https://github.com/openai/CLIP/archive/eaa22acb90a5876642d0507623e859909230a52d.zip",
-  "compel==0.1.7",
+  "compel @ https://github.com/damian0815/compel/archive/no_max_token_limit.zip",
```

Comment:
this doesn't work with pypi releases (so with this requirement we could not release the package to pypi)

Reply:
if you read the description you'll see that i'm planning ot push compel to pypi once this is confirmed working

Reply:
just a layer of security to prevent a merge to main 😅